src/js/engagement_view/tsconfig.json

```diff
@@ -25,5 +25,5 @@
   },
   "include": [
     "src/**/*.ts"
-, "src/tests/graphDisplay.test.tsx", "src/setupProxy.tsx"  ]
+, "src/tests/graphDisplay.test.tsx" ]
```

Comment:
Does the new `setupProxy.js` file get magically picked up somehow? Is the name special?

Reply:
Yeah, unfortunately. And so it's worth me mentioning this somewhere.

I think I'll document a) this behavior, and b) how to use it with Grapl in the README for this project. I'll add that to this PR.

Reply:
🤢 Ugh magic

Thanks for documenting these horrors for the rest of us :bow:

Reply:
Okay, I decided against updating the README for now, but there's a bunch of info in src/setupProxy.js.

Reply:
Nice addition; thanks for that 👍